src/Utopia/Messaging/Adapters/SMS/Vonage.php

```diff
@@ -48,7 +48,7 @@ protected function process(SMS $message): string
             body: \http_build_query([
                 'text' => $message->getContent(),
                 'from' => $message->getFrom(),
-                'to' => \implode(',', $to),
+                'to' => $to[0], //\implode(',', $to),
```

Comment:
Why do we force this to a single number?

Reply:
via the documentation: https://developer.vonage.com/en/api/sms?source=messaging

The API param for `to` says it's a string and the format of 1 number.